tests/api/test_aggregates.py

```diff
@@ -63,12 +68,14 @@ async def test_get_aggregates(
     assert fixture_aggregate_messages  # To avoid unused parameter warnings
 
     address = ADDRESS_1
-    aggregates = await get_aggregates_expect_success(ccn_api_client, address)
+    aggregates = await get_aggregates_expect_success(ccn_api_client, address, "True")
 
     assert address == aggregates["address"]
     assert aggregates["data"]["test_key"] == {"a": 1, "b": 2}
     assert aggregates["data"]["test_target"] == {"a": 1, "b": 2}
     assert aggregates["data"]["test_reference"] == {"a": 1, "b": 2, "c": 3, "d": 4}
+    assert aggregates["info"]["test_reference"] is not None
```

Comment:
Could you do more than just checking if the key is present? Like, compare to the content of the aggregate message.